src/main/java/org/ow2/proactive/catalog/report/TableDataBuilder.java

```diff
@@ -129,9 +163,10 @@ private String getProjectName(CatalogObjectMetadata catalogObject) {
     private String getDescription(CatalogObjectMetadata catalogObject) {
         return catalogObject.getMetadataList()
                             .stream()
-                            .filter(metadata -> metadata.getKey().equals("description"))
+                            .filter(metadata -> metadata.getLabel().equals("General") &&
+                                                metadata.getKey().equals("description"))
                             .map(metadata -> metadata.getValue())
-                            .map(description -> description.replaceAll("\n", "").replace("\r", "").replace("\t", ""))
+                            .map(description -> description.replaceAll("\n", " ").replace("\r", "").replace("\t", ""))
```

Comment:
Why do you need to use `replaceAll` for the description? It does the same thing as `replace` but takes a regex as first parameter. I think it can be handled the same as `\r` and `\t`.